src/api/serversStatus.ts

```diff
@@ -0,0 +1,23 @@
+import cp from 'child_process'
+import express from 'express'
+import environment from '../helpers/environment'
+import util from 'util'
+
+const statusRouter = express.Router()
+const exec = util.promisify(cp.exec)
+const { isProduction } = environment
+
+statusRouter.get('/', async (req, res) => {
+  try {
+    if (isProduction()) {
+      const data = await exec('su - myproxy -c "pm2 jlist"')
+      res.json(data)
```

Comment:
could you massage the data so the data output is predictable and easy to follow? Current code, new devs has no idea what `data` shape looks like. Instead:
```
const result = data.map(.....)
res.json(result)
```